### PR TITLE
Remove use of deprecated API

### PR DESCRIPTION
**Pull request by erlingrj**
Support this PR https://github.com/lf-lang/reactor-c/pull/148

### CODE REVIEW

File: Python/src/DigitalTwin/DoubleUnlock/DoubleUnlockDemo.lf [hunk 1]
```diff
@@ -96,7 +96,7 @@ reactor DoubleUnlockKeyFob(auto_lock_duration(5)) {
                     press_lock.schedule(0)
                 elif key == ord("u"):
                     press_unlock.schedule(0)
-            request_stop()
+            lf_request_stop()
```
Comment (lhstrh) [inline Python/src/DigitalTwin/DoubleUnlock/DoubleUnlockDemo.lf:99]: We have not regression tests here, but based on what I saw in `lingua-franca`, we should not make these changes in the Python target...

**Comment by erlingrj** [comment]
You are right. But I made those changes to `lingua-franca` and it didn't cause any test fail. That is surprising.

**Comment by erlingrj** [comment]
Ah, you have already addressed it. Thanks

File: C/src/experimental/ModalModels/BehaviorTrees/robohub_example_simple.lf [hunk 2]
```diff
@@ -105,12 +105,12 @@ main reactor {
...
+        lf_request_stop();
     =}
     
     reaction(robot.failure) {=
         printf("Utter failure\n");
-        request_stop();
+        lf_request_stop();
     =}
 
 }
```
Comment (lhstrh) [inline C/src/experimental/ModalModels/BehaviorTrees/robohub_example_simple.lf:116]:
```suggestion
}

```

**Review by lhstrh — APPROVED** [review]

**Pull request merged**